src/test/java/dynamo/RidesDaoTest.java

```diff
@@ -1,5 +1,60 @@
 package dynamo;
 
+import java.util.Arrays;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Table;
+import com.amazonaws.services.dynamodbv2.model.AttributeDefinition;
+import com.amazonaws.services.dynamodbv2.model.KeySchemaElement;
+import com.amazonaws.services.dynamodbv2.model.KeyType;
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughput;
+import com.amazonaws.services.dynamodbv2.model.ScalarAttributeType;
+
 public class RidesDaoTest {
+	private AmazonDynamoDB dbClient;
+	private DynamoDB dynamoDB;
+	public String tableName = "RideTableTest";
+
+	public void createTable() {
+		dbClient = AmazonDynamoDBClientBuilder.standard().withRegion(Regions.US_WEST_1).build();
+		dynamoDB = new DynamoDB(dbClient);
+
+		String tableName = "RideTableTest"; // Name for Table

```

Comment:
We probably don't want this here anymore since we have a member variable for it.